src/all/nijiero/src/eu/kanade/tachiyomi/extension/all/nijiero/Nijiero.kt

```diff
@@ -0,0 +1,148 @@
+package eu.kanade.tachiyomi.extension.all.nijiero
+
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.source.model.Filter
+import eu.kanade.tachiyomi.source.model.FilterList
+import eu.kanade.tachiyomi.source.model.Page
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.source.model.UpdateStrategy
+import eu.kanade.tachiyomi.source.online.ParsedHttpSource
+import okhttp3.HttpUrl.Companion.toHttpUrl
+import okhttp3.Request
+import org.jsoup.nodes.Document
+import org.jsoup.nodes.Element
+import java.text.SimpleDateFormat
+import java.util.Locale
+
+class Nijiero : ParsedHttpSource() {
+    override val name = "Nijiero"
+    override val lang = "all"
+    override val supportsLatest = true
+
+    override val baseUrl = "https://www.nijiero-ch.com"
+
+    override val client = network.client.newBuilder().followRedirects(true).build()
```

Comment:
```suggestion
    override val client = network.cloudflareClient
```
default client already follows redirect